package/python-kiwi-spec-template

```diff
@@ -66,11 +66,18 @@
 %define distro rhel-07.0
 %endif
 
+# Fedora
+# use the rhel templates for CentOS, too
+%if 0%{?fedora_version}
+%define distro fedora-25.0
```

Comment:
Currently, there are only tags for Fedora 25 in the python-kiwi-boot-packages so I use them also for Fedora 26.